### PR TITLE
Geth v1.9.11

### PR DESCRIPTION
**Pull request by yaoandrew**
This is still failing for a few platforms based on some linting errors. Should I continue to try and fix these to get to all green? How far do we take this fork usually?

### CODE REVIEW

**Comment by yaoandrew** [comment]
This is still failing for a few platforms based on some linting errors. Should I continue to try and fix these to get to all green? How far do we take this fork usually?

File: cmd/faucet/website.go [hunk 1]
```diff
@@ -105,27 +105,27 @@ func Asset(name string) ([]byte, error) {
...
 
 // AssetString returns the asset contents as a string (instead of a []byte).
-func AssetString(name string) (string, error) {
-	data, err := Asset(name)
-	return string(data), err
-}
+//func AssetString(name string) (string, error) {
+//	data, err := Asset(name)
+//	return string(data), err
+//}
```
Comment (yaoandrew) [inline cmd/faucet/website.go]: Linter was complaining about these not being called, but the top of the file says do not edit... this feels 😬 💀

File: eth/config.go [hunk 2]
```diff
@@ -166,6 +166,6 @@ type Config struct {
 
 	// MuirGlacier block override (TODO: remove after the fork)
 	OverrideMuirGlacier *big.Int `toml:",omitempty"`
-	
+	// Added by EE
```
Comment (yaoandrew) [inline eth/config.go:169]: The linter also complained about this line being blank. 😕

Comment (elizabethengelman) [reply eth/config.go:169]: I think this is safe to delete... not sure why that blank was added 😬

File: cmd/faucet/website.go (hunk 1 above)
Comment (elizabethengelman) [reply cmd/faucet/website.go]: weird! I think not editing it makes sense... I wonder if the linter tests are passing on geth's master right now. 🤔

Comment (yaoandrew) [reply cmd/faucet/website.go]: [This](https://travis-ci.org/ethereum/go-ethereum/builds/651941757) is the build for the commit with the 1.9.11 label. Its failing for an unrelated reason. 🤷‍♂ Master is failing for a different linting reason

**Comment by elizabethengelman** [comment]
[.golangci.yml](https://github.com/ethereum/go-ethereum/blob/master/.golangci.yml) on geth master is slightly different than what we have in this branch - looks 
```
    - path: cmd/faucet/
      linters:
        - deadcode
```  
was added so that the linters would pass

**Comment by elizabethengelman** [comment]
Also, when running git diff between this branch and upstream v1.9.11 it looks like we're missing 
```
skip-files:
    - core/genesis_alloc.go
``` 
from golanci.yml. Probably not a huge deal, just wanted to point it out.

**Review by elizabethengelman — APPROVED** [review]

**Pull request merged**